docs/msbuild/item-element-msbuild.md

```diff
@@ -73,6 +73,8 @@ Now, however, you can pass the `Version` metadata as an attribute, such as in th
 |`Exclude`|Optional attribute.<br /><br /> The file or wildcard to exclude from the list of items.|
 |`Condition`|Optional attribute.<br /><br /> The condition to be evaluated. For more information, see [Conditions](../msbuild/msbuild-conditions.md).|
 |`Remove`|Optional attribute.<br /><br /> The file or wildcard to remove from the list of items.<br /><br />|
+|`MatchOnMetadata`|Optional attribute.<br /><br /> Modifies `Remove` attributes that reference other items to match on the specified metadata names instead of matching on the referenced items' values. <br /><br />  This attribute is valid only if it's specified together with a `Remove` attribute that only contains references to other items (e.g. `Remove="@(Compile);@(Content)"`). More details in [Items](../msbuild/msbuild-items.md).
```

Comment:
```suggestion
|`MatchOnMetadata`|Optional attribute.<br /><br /> Modifies `Remove` attributes that reference other items to match on the specified metadata names instead of matching on the referenced items' values. <br /><br /> This attribute is valid only if it's specified together with a `Remove` attribute that only contains references to other items (for example, `Remove="@(Compile);@(Content)"`). More details in [Items](../msbuild/msbuild-items.md).
```